transform/snowflake-dbt/models/hightouch/freemium/enterprise_trial_requests/etr_campaignmember_insert.sql

```diff
@@ -0,0 +1,54 @@
+{{config({
+    "schema": "hightouch",
+    "materialized": "view",
+    "tags":["hourly","freemium"]
+  })
+}}
+with existing_members as (
+    select
+        campaignmember.sfid,
+        campaignmember.email,
+        campaignmember.dwh_external_id__c,
+        row_number() over (partition by campaignmember.email order by createddate desc) as row_num
+    from {{ ref('campaignmember') }}
+    where campaignmember.campaignid = '7016u0000002OHjAAM'
+    -- where campaignmember.campaignid = '7013p000001U2GbAAK'
+), existing_leads as (
+    select
+        lead.sfid,
+        lead.email,
+        lead.dwh_external_id__c,
+        row_number() over (partition by lead.email order by createddate desc) as row_num
+    from {{ ref('lead') }}
+), existing_contacts as (
+    select
+        contact.sfid,
+        contact.email,
+        contact.dwh_external_id__c,
+        row_number() over (partition by contact.email order by createddate desc) as row_num
+    from {{ ref('contact') }}
+), campaignmembers_to_sync as (
+    SELECT
+        existing_leads.sfid,
+        customers_with_freemium_subs.email,
+        customers_with_freemium_subs.company,
+        COALESCE(existing_contacts.dwh_external_id__c, customers_with_freemium_subs.contact_external_id) as contact_external_id,
+        COALESCE(
+            existing_leads.dwh_external_id__c,
+            UUID_STRING(
+                '78157189-82de-4f4d-9db3-88c601fbc22e',
+                customers_with_freemium_subs.customer_id || customers_with_freemium_subs.email)
+        ) AS lead_external_id,
+        '7016u0000002OHjAAM' as sandbox_campaign_id,
+        '7013p000001U2GbAAK' as campaign_id,
+        'Responded' as campaignmember_status
+    FROM {{ ref('customers_with_freemium_subs_without_filter') }} as customers_with_freemium_subs
```

Comment:
why not name `as customers_with_freemium_subs` -> `as customers_with_freemium_subs_without_filter` to be consistent?

Reply:
this was part of previous logic, which I have updated in new models.